### PR TITLE
Feature/functionality of classes

### PR DESCRIPTION
**Pull request by Wperricone**
#### What's this PR do?
- creates functionality and tests for recipe card and recipe repository

#### Where should the reviewer start?
-with the recipe repository class and the recipe card class

#### How should this be manually tested?
- npm test in command line interface

#### Questions:
- why is sea salt so expensive?

### CODE REVIEW

File: src/classes/RecipeCard.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+class RecipeCard {
+  constructor(recipe) {
+    this.recipe = recipe;
+    this.name = recipe.name;
+    this.ingredients = recipe.ingredients;
+    this.image = recipe.image;
+    this.id = recipe.id;
+    this.tags = recipe.tags;
+    this.instructions = recipe.instructions;
+
```
Comment (sted1994) [inline src/classes/RecipeCard.js:10]: Can we remove this blank line in the next push.

File: test/RecipeCard-test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import RecipeRepository from '../src/classes/RecipeRepository';
+import RecipeCard from '../src/classes/RecipeCard';
+const { recipeData } = require('../src/data/recipes');
+const { ingredientsData } = require('../src/data/ingredients');
+
+
```
Comment (sted1994) [inline test/RecipeCard-test.js:7]: Can we remove this blank line in the next push.

File: test/RecipeCard-test.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+describe('Recipe Card', () => {
+  let recipeCard;
+    beforeEach(() => {
+      recipeCard = new RecipeCard(recipeData[0])
+
```
Comment (sted1994) [inline test/RecipeCard-test.js:12]: Can we remove this blank line in the next push.

File: test/RecipeCard-test.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    });
+
+  it('Should be a function', () => {
+    expect(RecipeCard).to.be.a('function');
+  });
+
+it('should create a recipe card', () => {
+
```
Comment (sted1994) [inline test/RecipeCard-test.js:20]: Can we remove this blank line in the next push.

**Review by sted1994 — APPROVED** [review]
Great work! Look forward to continuing.

File: test/RecipeRepository-test.js [hunk 5]
```diff
@@ -1,8 +1,30 @@
...
 
-describe('Recipe', () => {
-  it('Should be a function', () => {
-    expect(RecipeRepository).to.be.a('function');
-  });
-})
\ No newline at end of file
+describe('Recipe Repository', () => {
+  let recipeList;
+    beforeEach(() => {
```
Comment (zwsaile) [inline test/RecipeRepository-test.js:8]: Lovin' the beforeEach usage

**Review by zwsaile — APPROVED** [review]

**Pull request merged**